src/v0/util/index.js

```diff
@@ -1681,12 +1681,7 @@ const validatePhoneWithCountryCode = (phone) => {
  * @param {*} Config
  * @returns
  */
-const isHybridModeEnabled = (Config) => {
-  if (isDefinedAndNotNull(Config.useNativeSDK) && isDefinedAndNotNull(Config.useNativeSDKToSend)) {
-    return Config.useNativeSDK && !Config.useNativeSDKToSend;
-  }
-  return false;
-};
+const isHybridModeEnabled = (Config) => Config.connectionMode === 'hybrid';
```

Comment:
DB configs moved this new value?

Reply:
no, `connectionMode` was already coming but we were not using it. now as we decided to check hybridmode condition based on `connectionMode` key we will use that one

Reply:
is it true for any destination?

Reply:
only the one who is in new form builder and this is only recommended to use for only new form builder destinations